lib/BridgeAbstract.php

```diff
@@ -142,7 +155,10 @@ public function setDatas(array $inputs){
 			if($time !== false
 			&& (time() - static::CACHE_TIMEOUT < $time)
 			&& (!defined('DEBUG') || DEBUG !== true)){
-				$this->items = $this->cache->loadData();
+				$cached = array();
```

Comment:
Right, I just overlike to declare a variable before resetting it, as you saw.
I will remove this line.